src/components/VideoGrid/index.jsx

```diff
@@ -0,0 +1,27 @@
+import React, { useContext } from 'react';
+import VideoCard from '../VideoCard';
+import { Grid, Row, Col } from './styled';
+import ThemeContext from '../../state/ThemeContext';
+
+const VideoGrid = ({ videos }) => {
+  const { stateTheme } = useContext(ThemeContext);
+  const { theme } = stateTheme;
+
+  return (
+    <Grid theme={theme} data-testid="grid">
+      <Row data-testid="row">
+        {videos.map((video) =>
+          video.id.videoId ? (
+            <Col key={video.id.videoId}>
+              <VideoCard video={video} />
+            </Col>
+          ) : (
+            ''
```

Comment:
`<Row>` tag is expecting that map returns JSX at each element, if the internal conditional prevents this, you could have awful warnings or even errors when rendering.  I suggest using :
1. `filter()` from arrays, or
2. return null instead of an empty string